docs/src/components/home/GoldSponsors.tsx

```diff
@@ -61,6 +61,20 @@ const GOLDs = [
     description: 'Horse community.',
     href: 'https://www.hoodiebees.com/?utm_source=MUI&utm_medium=referral&utm_content=homepage',
   },
+  {
+    src: 'https://images.opencollective.com/sumatosoft_company/0b78570/avatar/40.png',
+    srcSet: 'https://images.opencollective.com/sumatosoft_company/0b78570/avatar/80.png 2x',
+    name: 'SunmatoSoft',
+    description: 'IT solutions and services to established companies that need tailored software.',
```

Comment:
I might take a bit too much space: 

<img width="415" alt="Screenshot 2021-11-29 at 12 26 07" src="https://user-images.githubusercontent.com/3165635/143859866-120225bb-980b-43a6-807a-899dc3461d85.png">

Maybe we could take 

```suggestion
    description: 'We help companies to digitalize their businesses.',
```